### ISSUE TITLE
Add YouTube to Learn line up on the Community Page on website

### ISSUE THREAD

**Issue by jendefig** [iterative/dvc.org#2603]
In the Learn portion of our Community page, I think we should add YouTube Channel to the sections with latest YouTube posts populating *or* most viewed YouTube videos.  And it should go at the end so Documentation -> DVC Blog -> User Content -> YouTube Channel.

@rogermparent @shcheklein

**Comment by shcheklein** [comment]
This is great, agreed that we need to promote our videos in a few places.

**Comment by rogermparent** [comment]
Sounds like a great idea to me as well!

There's a few plugins for sourcing from YouTube, one is [gatsby-source-youtube-v3](https://www.gatsbyjs.com/plugins/gatsby-source-youtube-v3/?=youtube) and the next is [gatsby-source-youtube-v2](https://www.gatsbyjs.com/plugins/gatsby-source-youtube-v2/?=youtube)

While dvc.org _is_ gatsby v2, I feel we should try the v3 plugin first since it's actively updated, and so should prove more resistant to any future changes of YouTube's API. If the v3 plugin depends on something our v2 site doesn't have, we can try the v2 version of the plugin.

I added @julieg18 to the assignees, just to keep her posted since she'll probably be working on this when time and workload allows.

**Comment by jendefig** [comment]
I'd like to add this to the Roadmap for website if possible for Q4 too. @julieg18

**Comment by julieg18** [comment]
> I'd like to add this to the Roadmap for website if possible for Q4 too. @julieg18

Sure! This should be doable for Q4 :)

**Comment by jendefig** [comment]
In the Learn Section as well, now that we will have the full course, we need to direct to the course as well.

**Comment by jendefig** [comment]
Reviving this discussion. @rogermparent @shcheklein!

Re: [this issue to drop Events](https://github.com/iterative/dvc.org/issues/3897) for the time being until we can figure a way to automate this.  If it's too hard to fit the YouTube feed into a fourth section in the Learn Header, then could we make another section for our YouTube videos to replace the Events section?

Is there a way to automatically feed the new YouTube videos to this section in the website?

**Comment by rogermparent** [comment]
Yup, I think the solutions discussed in this thread could pretty easily power what you're describing. We can put this back on the radar, but should it come before or after #3898?

**Comment by rogermparent** [comment]
Looks like `gatsby-source-rss-feed` with YouTube's built-in RSS feed works out of the box, and even has thumbnail urls we can use!
![image](https://user-images.githubusercontent.com/9111807/186818132-c74151d3-fecc-4994-84a5-c67c6469afd1.png)

**Comment by jendefig** [comment]
> but should it come before or after #3898

I would think after as I'm assuming it would be easier to just do add YouTube once in iterative.ai then potential complications of both?
